core/tests/player_test.py

```diff
@@ -60,5 +61,30 @@ def test_add_species__removes_discard_card_from_player_hand_cards(self):
             self.player.hand_cards
         )
 
-    # Todo: write add_species position left and right tests.
```

Comment:
Doesn't PyCharm normally style this as "TODO:" rather than "Todo:"? Or I guess it just works with both?

Reply:
It works with both. Pycharm also highlights ```#Fixme```